repos/system_upgrade/common/libraries/dnfplugin.py

```diff
@@ -337,22 +339,28 @@ def perform_transaction_install(target_userspace_info, storage_info, used_repos,
         # communicate with udev
         cmd_prefix = ['nsenter', '--ipc=/installroot/proc/1/ns/ipc']
 
+        disable_plugins = []
+        if plugin_info:
```

Comment:
Nit: this can be moved into a function to be reused at L395 and L427.

Reply:
yes, but let's keep it already as it is in this phase. waiting for the test results. I am in favor to merge this one when tests pass